records/generator.go

```diff
@@ -150,41 +150,44 @@ func (rg *RecordGenerator) findMaster(masters ...string) (state.State, error) {
 
 	// Check if ZK leader is correct
 	if leader != "" {
-		logging.VeryVerbose.Println("Zookeeper says the leader is: ", leader)
+		logging.VeryVerbose.Println("The leader is: ", leader)
 		ip, port, err := getProto(leader)
 		if err != nil {
 			logging.Error.Println(err)
+		} else {
+			if sj, err = rg.loadWrap(ip, port); err == nil && sj.Leader != "" {
```

Comment:
True :) I just did not want to change the logic here too much.

Considering the changes in `loadWrap`, `sj.Leader != ""` will always be True as long as `err==nill`.